client/state/site-rename/actions.js

```diff
@@ -0,0 +1,96 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import page from 'page';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	SITE_RENAME_REQUEST,
+	SITE_RENAME_REQUEST_FAILURE,
+	SITE_RENAME_REQUEST_SUCCESS,
+} from 'state/action-types';
+import { errorNotice, successNotice } from 'state/notices/actions';
+import { domainManagementEdit } from 'my-sites/domains/paths';
+import { requestSite } from 'state/sites/actions';
+
+// @TODO proper redux data layer stuff for the nonce
+function fetchNonce( siteId ) {
+	return wpcom.undocumented().getRequestSiteRenameNonce( siteId );
+}
+
+export const getErrorNotice = message =>
+	errorNotice( message, {
+		id: 'siteRenameUnsuccessful',
+		duration: 5000,
+		showDismiss: true,
+		isPersistent: true,
+	} );
+
+const dispatchErrorNotice = ( dispatch, error ) =>
+	dispatch(
+		getErrorNotice(
+			// @TODO translate copy once finalised
+			error.message || "Sorry, we we're unable to complete your domain change. Please try again."
```

Comment:
`we we're` :)

Reply:
😬

Reply:
🙈

Reply:
Addressed by @jblz in https://github.com/Automattic/wp-calypso/pull/22661/